test/export/test_export.py

```diff
@@ -10092,30 +10092,30 @@ def forward(self, x):
         )
 
     def test_preserve_cia_op(self):
```

Comment:
I've updated this test to use Trilinear3d to preserve the original intent of the test. Since Bilinear2d is core, it is always preserved (and also the test fails, given that it can't pop the decomp). Trilinear3d is not core and is composite implicit autograd.